src/Utils.h

```diff
@@ -39,7 +39,7 @@
 
 [[gnu::unused]] static String prettyPrintFileSystemType(FileSystems f){
     if(f == 0) return "FAT";
-    else if(f == 1) return "LitlleFS";
+    else if(f == 1) return "LittleFS";
```

Comment:
This fix is also included in https://github.com/arduino-libraries/Arduino_UnifiedStorage/pull/44